lib/drives/pages/drive_detail_page.dart

```diff
@@ -327,8 +361,15 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
           child: Material(
             color: Colors.transparent,
             child: InkWell(
-              //TODO go to chat
-              onTap: () {},
+              onTap: () => Navigator.push(
```

Comment:
This doesn't work for either of us. It shows an empty chat every time and if you try to chat, the "Something went wrong" snackbar shows

Reply:
Okay that's weird.

Reply:
because it's working for me